variables.tf

```diff
@@ -632,6 +632,16 @@ variable "runners_services_volumes_tmpfs" {
   default = []
 }
 
+variable "runners_docker_services" {
```

Comment:
Can you add a description?

Reply:
Added, but the pre-commit didn't add this var to the readme. Did I miss a step?

Reply:
I had removed recently terraform docs from the pre commit, but looks like I need to update docs. Terraform docs is now running in CI on the develop branch.